godel_msgs/msg/ScanPlanParameters.msg

```diff
@@ -0,0 +1,12 @@
+# Path Parameters
+float64 scan_width           	# (m) actual width of scan tool
```

Comment:
Width of the tool, or of the scanning area? `scan_width` to me could also be the width of the 'beam' so to say.